src/client/src/translations/en.json

```diff
@@ -115,12 +115,12 @@
   "projectName.projectName": "Project Name",
   "projectName.projectNameTitle": "Project Name:",
   "projectNameError.invalidRegex": "Name may only contain letters, numbers, dashes or underscores",
-  "review.frameworks": "3. Frameworks",
-  "review.pages": "4. Pages",
-  "review.projectType": "2. Project Type",
-  "review.reviewAndGenerate": "6. Your project summary.",
-  "review.services": "5. Services",
-  "review.welcome": "1. Welcome",
+  "review.frameworks": "Frameworks",
```

Comment:
You modify review.frameworks.
![image](https://user-images.githubusercontent.com/32103099/58519558-31a2df80-8182-11e9-8fa3-f2bebd442368.png)
The actual thing is untouched though :( am I missing something here?

Reply:
Weird i tried doing the update command but its not detecting it for some reason.

Reply:
Turns out its because react intl doesn't detect string changes by default. deleting the file "en.json" and adding it back by running ```npm run manage:translations``` will work.

Reply:
I see.. I saw @piotrmark 's comment too and ran the command. Seems like there's some changes to whitelist_en.json as well. Is that expected?

Reply:
Yeah both are fine, in fact delete both and let the thing regenerate itself. They are both automated things so regeneration won't do any damage!

Reply:
When we actually do full translations of such (after clint sends it off for translation that is when you want to be more careful with the translation jsons, but right now they're mutable).

Reply:
Also, I followed the steps you (@SahilTara) mentioned, and it seems like other things not related to this issue are changed. Should they be added to this PR as well? 

Here's an example:
![image](https://user-images.githubusercontent.com/20333350/58520465-d61a2700-816c-11e9-8ac6-c25ce5f0b5f3.png)

Reply:
Honestly for this auto generated file I would say yes its okay, because its 100% auto-generated and you have no real control over it. This also exposes that this was due for an update for a while. @piotrmark can correct me if i'm wrong here. Worst case I would try just deleting the suspect lines from the whitelist and en.json, and seeing if they regenerate if you're worried about it.

Reply:
Actually you know what yeah just do git reset --hard HEAD then just remove the lines you modified and see if they regenerate. that seems like a better option.

Reply:
Thanks Sahil! I manually deleted the en.json file and re-ran `npm run manage:translations`, seems like the lines relevant to this PR are generated properly. 
Your suggestion sounds good; I'll create a separated issue to update the rest of the en.json file then.